ql/src/java/org/apache/hadoop/hive/ql/metadata/Hive.java

```diff
@@ -4204,9 +4223,11 @@ public Set<Partition> getAllPartitionsInBatches(Table tbl, int batchSize, int de
       public Void execute(int size) throws HiveException {
         try {
           result.clear();
-          new PartitionIterable(Hive.get(), tbl, null, size).forEach(result::add);
+          new PartitionIterable(Hive.get(), tbl, partialPartitionSpec, size, isAuthRequired, userName,
+                  groupNames).forEach(result::add);
           return null;
         } catch (HiveException e) {
+          LOG.error("Unable to get Partition List", e);
```

Comment:
Since you are anyway throwing e which will be logged anyway, is there any point to log here?

Reply:
In fact, you can get rid of entire try catch as anyway method signature has "throws HiveException"

Reply:
changed